decompiler/config/all-types.gc

```diff
@@ -19443,47 +19443,47 @@
 
 ;; - Functions
 
-(define-extern line-in-view-frustum? function)
+(define-extern line-in-view-frustum? (function vector vector symbol)) ;; TODO - not confirmed
 (define-extern process-drawable-random-point! (function process-drawable vector vector))
 (define-extern process-drawable-from-entity! (function process-drawable res-lump none))
-(define-extern cam-launcher-long-joystick function)
+(define-extern cam-launcher-long-joystick (function vector :behavior camera-slave))
 (define-extern hide-hud-quick (function none))
-(define-extern command-get-process (function basic process-drawable process))
-(define-extern camera-change-to (function string int symbol none)) ;; TODO - not confirmed yet
-(define-extern camera-look-at (function process int none)) ;; TODO - process is likely a target
-(define-extern camera-pov-from function)
-(define-extern command-get-trans function)
-(define-extern manipy-init (function vector entity skeleton-group vector none :behavior manipy))
-(define-extern part-tracker-notify function)
+(define-extern command-get-process (function pair process-drawable process))
+(define-extern camera-change-to (function string int symbol symbol))
+(define-extern camera-look-at (function pair uint process :behavior camera-tracker))
+(define-extern camera-pov-from (function pair uint process :behavior camera-tracker))
+(define-extern command-get-trans (function pair vector vector))
```

Comment:
These don't actually take pairs. I've seen them be called with processes and strings in their args. This should just be `object`, looks like they could put anything.